package ex_20231101.userDefineException_Ex01;

import java.util.Scanner;

public class NumberMenu {
	public static void main(String[] args) {
		
	}
	
	public void menu() { }
		
		
		/*   설명 : 사용자에게 두 정수를 입력 받아 NumberController의 checkDouble()에 
			 매개변수로 넘겨주어 반환 값을 출력함
			 이 때 checkDouble()메소드를 호출하는 부분부터 try~catch를 이용하여 예외 처리
			 
			 정수1 : 55
			 정수2 : 5
			 55은(는) 5의 배수인가 ? true 
			 
			 
			 Bank bank = new Bank();
		
		// 입금
		bank.deposit(10000);
		System.out.println("현재 잔액 : " + bank.getBalance() + "원");
		
		// 출금
		try {
			bank.withdraw(9700);
			System.out.println("현재 잔액 : " + bank.getBalance() + "원");
		} catch (UserException e) {
			System.out.println(e.getMessage());
		}
		*/

}
